sound/soc/sof/intel/hda-dsp.c

```diff
@@ -398,6 +398,9 @@ static int hda_dsp_update_d0i3c_register(struct snd_sof_dev *sdev, u8 value)
 	snd_sof_dsp_update8(sdev, HDA_DSP_HDA_BAR, chip->d0i3_offset,
 			    SOF_HDA_VS_D0I3C_I3, value);
 
+	/* Wait for about 30 us to propagate I3 transition */
```

Comment:
commit message: 'wait according to spec advice'? Which spec? PCI or hardware programming sequence?

Reply:
thanks!